examples/example_host.cpp

```diff
@@ -4,6 +4,7 @@
 
 /// @file
 /// Example implementation of an EVMC Host.
+/// TODO: Doesn't support TXCREATE transaction initcodes.
```

Comment:
Not sure it makes sense to support it, `example.c` only sends `EVMC_CALL` message anyway.

Reply:
feel free to drop the comment like this, if you feel like nothing needs to be done. I was very generous when adding them in